.github/workflows/canister-tests.yml

```diff
@@ -481,6 +481,8 @@ jobs:
           until [ -f ~/npm-ci-status ]; do sleep 1; done
           exit $(cat ~/npm-ci-status)
 
+      - name: Build sig-verifier
+        run: npm run --workspace ./src/sig-verifier-js build
       - name: Run sig-verifier tests
         run: npm run --workspace ./src/sig-verifier-js test
```

Comment:
@frederikrothenberger this was the only place where the library is used, to test it.

Instead of relying on `npm ci` to also build it, I added the specific command to build it. I hope that's ok for you.

Reply:
Yes, definitely! Thanks a lot 👍